Assets/VRM/UniGLTF/Scripts/Format/glTFBuffer.cs

```diff
@@ -24,6 +24,11 @@ public void OpenStorage(IStorage storage)
             */
         }
 
+        public glTFBuffer()
+        {
+
+        }
+
         public glTFBuffer(IBytesBuffer storage)
```

Comment:
```suggestion
        public glTFBuffer(IBytesBuffer storage) : this()
```

Reply:
I think that it is not necessary

Reply:
Oops, sorry. Other fields are not initialized by the default constructor. You are right.